src/components/search/Search.ts

```diff
@@ -0,0 +1,236 @@
+import { html, nothing } from "lit"
+import { classMap } from "lit/directives/class-map.js"
+import { live } from "lit/directives/live.js"
+import { createRef, Ref, ref } from "lit/directives/ref.js"
+import { property, state } from "lit/decorators.js"
+
+import { LeuElement } from "../../lib/LeuElement.js"
+import { LeuIcon } from "../icon/Icon.js"
+import { LeuButton } from "../button/Button.js"
+
+import styles from "./search.css"
+
+export const SEARCH_SIZES = {
```

Comment:
Die definierten Konstante würde ich in die Story verschieben, da sie nur dort verwendet wird. Bin das jetzt auch bei den anderen Components am nachziehen